Automattic/wp-calypso
# Fix: Non-tabbed plans looks ugly for some currencies

#### Changes proposed in this Pull Request

The non-tabbed plans presentation looks ugly for some currencies including Japanese Yen.
<img width="700" alt="요금제_‹_사이트_제목_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/56117200-89a2cf80-5fa2-11e9-9a18-ff3313de64a3.png">

This PR fixes the styles so the plans would look better.
<img width="800" alt="요금제_‹_사이트_제목_—_WordPress_com_및_Slack_-_A8C" src="https://user-images.githubusercontent.com/212034/56117285-bb1b9b00-5fa2-11e9-9d66-44c21cd16607.png">
<img width="800" alt="요금제_‹_사이트_제목_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/56117288-bb1b9b00-5fa2-11e9-93e9-5666836b281d.png">

#### Testing instructions



* Create a new website.
* Run the following command in the browser console to activate a/b test. Note that the Calypso-side a/b test will be removed once this PR gets approved or rejected.
   ```
   localStorage.setItem('ABTests', '{"plansNoTabs_20190415": "test"}');
   ```
* Visit `/plans/YOUR_SITE` page. You will be able to see the scrollable plans.
